Jenkinsfile

```diff
@@ -0,0 +1,54 @@
+pipeline {
+    agent none
+    environment {
+        projectName = 'DDF'
+    }
+    stages {
+        stage('Setup') {
+            steps{
+                notifySlack('good', "STARTED: ${JOB_NAME} ${BUILD_NUMBER} ${BUILD_URL}")
+            }
+        }
+        stage('Parallel Build') {
+            // TODO refactor this stage from scripted syntax to declarative syntax to match the rest of the stages - https://issues.jenkins-ci.org/browse/JENKINS-41334
+            steps{
+                parallel(
+                    linux: {
+                        node('linux-large') {
+                            timeout(time: 3, unit: 'HOURS') {
+                                checkout scm
+                                withMaven(maven: 'M3', mavenSettingsConfig: 'codice-maven-settings') {
+                                    sh 'mvn clean install'
+                                }
+                            }
+                        }
+                    }, windows: {
+                        node('proxmox-windows'){
+                            timeout(time: 3, unit: 'HOURS') {
+                                checkout scm
+                                withMaven(maven: 'M3', mavenSettingsConfig: 'codice-maven-settings') {
+                                    bat 'mvn clean install'
+                                }
+                            }
+                        }
+                    }
+                )
+            }
+        }
+    }
+    post {
+        success {
+            notifySlack('good', "SUCCESS: ${JOB_NAME} ${BUILD_NUMBER}")
+        }
+        failure {
+            notifySlack('#ea0017', "FAILURE: ${JOB_NAME} ${BUILD_NUMBER}. See the results here: ${BUILD_URL}")
+        }
+        unstable {
+            notifySlack('#ffb600', "UNSTABLE: ${JOB_NAME} ${BUILD_NUMBER}. See the results here: ${BUILD_URL}")
+        }
+    }
+}
+
+def notifySlack(String color, String message) {
```

Comment:
we can probably just change to calling `slackSend` directly since all this does at the moment is proxy that call. I think originally this method had more logic in it to determine the message that should be sent based on the build status